.github/pull_request_template.md

```diff
@@ -1,6 +1,12 @@
 ## Requirements
 
 - [ ] This PR has a title that briefly describes the work done, including the ticket number if there is a ticket.
+- [ ] We use PR titles to determine version bumps. PR titles should start with something in parentheses, or the word BREAKING. Examples of good PR titles:
+<br /> “(docs) Add to contributing docs”
+<br />“(fix) Console error when visiting allergies page”
+<br />“(feat) Add search bar to medications widget”
+<br />“(refactor) Tidy the dashboard implementation”
+<br />“BREAKING: New left nav system”
```

Comment:
- [ ] This PR has a title that briefly describes the work done, including the ticket number if there is a ticket. Make sure your PR title includes a [conventional commit](https://o3-dev.docs.openmrs.org/#/getting_started/contributing?id=your-pr-title-should-indicate-the-type-of-change-it-is) label.

Reply:
Reckon linking to the Contributing guide section about conventional commit labels should suffice.